Dockerfile

```diff
@@ -16,17 +16,18 @@ RUN set -ex && pip3 install pipenv --upgrade
 
 # Copy in the code
 COPY *.py /app/
+COPY nerium/ /app/nerium/
 COPY Pipfile /app/
 WORKDIR /app
 VOLUME /app/query_files
 # Install requirements
 COPY --from=gcsfuse-build  /go/bin/gcsfuse /usr/local/bin/
 RUN pipenv install --system
 
-# $db_driver may be one of "postgres" or "mysql"
-ARG db_driver
 
-RUN if [ "$db_driver" = "postgres" ]; then apk add --no-cache py3-psycopg2; fi
-RUN if [ "$db_driver" = "mysql" ]; then pipenv install --system pymysql; fi
+RUN apk add --no-cache py3-psycopg2
+RUN pipenv install --system pymysql
```

Comment:
If we're going to always load both of these, we should consolidate with the main install steps earlier.